docs/guide/idp-app-token.md

```diff
@@ -0,0 +1,39 @@
+# Fake IdP SAML App Canarytoken
```

Comment:
do we think this guide would be better with some images? im not sure if we do this for tokens generally but for these nitty gritty do this, then that, then over here click this kinda things i feel like they're helpful?